corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -955,9 +955,9 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
                     break;
                 }
             }
-            // corresponds to validateFile check in Formplayer
-            // any changes made here should also be made in Formplayer
-            if (badExtension && badMime) {
+            // similar to validateFile check in Formplayer
+            // though this check is more strict than the one in Formplayer
```

Comment:
@Robert-Costello can we also change the FP validation to do the same validation. Thanks!

Reply:
thanks!